src/main/java/Duke.java

```diff
@@ -1,10 +1,136 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+import java.util.Set;
+import task.DeadlineTask;
+import task.EventTask;
+import task.Task;
```

Comment:
I really like how you only import the relevant libraries that are needed! Instead of using * import!